src/components/states/instructions.js

```diff
@@ -6,22 +6,40 @@ AFRAME.registerComponent('instructions', {
     this.startButtonEl = document.querySelector('.start-button');
     this.startButtonEl.querySelector('span').innerHTML = 'Loading';
     this.startButtonEl.querySelector('span').classList.add('loading');
-    this.onClicked = this.onClicked.bind(this);
+    this.onStartButtonClicked = this.onStartButtonClicked.bind(this);
     this.loading = true;
     this.el.sceneEl.addEventListener('loaded', this.downloadDance.bind(this));
   },
 
+  handleDanceData: function (data) {
+    this.danceData = data;
+
+    var sceneEl = this.el.sceneEl;
+    var inVR = sceneEl.is('vr-mode');
+
+    if (inVR) {
+      this.onStartButtonClicked();
+    } else {
+      var self = this;
+      window.addEventListener('vrdisplayactivate', function () {
+        if (sceneEl.getAttribute('game-state').state === 'instructions') {
+          self.onStartButtonClicked();
+        }
+        //@todo remove eventlistener?
```

Comment:
We hide the whole UI when playing but what I mean there is if we should remove the `vrdisplayactivate` listener too.
Probably it doesn't matter as once we're presenting we can't go back to the instructions so the case of that listener being triggered several times won't happen either